src/components/views/rooms/Stickerpicker.js

```diff
@@ -379,14 +380,21 @@ export default class Stickerpicker extends React.Component {
     render() {
         let stickerPicker;
         let stickersButton;
+        const className = classNames(
+            "mx_MessageComposer_button",
+            "mx_MessageComposer_stickers",
+            "mx_Stickers_hideStickers",
+            "mx_RightPanel_headerButton_highlight",
```

Comment:
this needs a comment or a more appropriate style - taking styles from the right panel is bad

Reply:
Made it's own highlight in _MessageComposer.scss